closure/goog/dom/forms_test.js

```diff
@@ -417,6 +417,26 @@ function testSetValueInput() {
   assertEquals('', goog.dom.forms.getValue(el));
 }
 
+function testGetValueMeter() {
```

Comment:
@shicks ok i think i've done as much as i can for this PR, let me know how you go